src/temp.js

```diff
@@ -0,0 +1,109 @@
+import removeAllChildNodes from './removeAllChildNodes';
+
+let tempType = [];
+if (localStorage.getItem('tempType') === null) {
+  localStorage.setItem('tempType', JSON.stringify([]));
+} else {
+  tempType = JSON.parse(localStorage.getItem('tempType'));
+}
+
+const fahCel = document.querySelector('.fahCel');
+let fahCel2 = '';
+tempType = JSON.parse(localStorage.getItem('tempType'));
+
+if (tempType[0] === undefined) {
+  fahCel.value = 'Get Celsius';
+  fahCel2 = 'imperial';
+} else {
+  if (tempType[0] === 'Get Fahrenheit') {
+    fahCel.value = 'Get Celsius';
+    fahCel2 = 'metric';
+  } else {
+    fahCel.value = 'Get Fahrenheit';
+    fahCel2 = 'imperial';
+  }
+  fahCel.value = tempType[0];// eslint-disable-line
+}
+
+const changeTemp = () => {
+  if (fahCel.value === 'Get Fahrenheit') {
+    tempType = JSON.parse(localStorage.getItem('tempType'));
+    tempType[0] = 'Get Fahrenheit';
+    fahCel.value = 'Get Celsius';
+    fahCel2 = 'imperial';
+    localStorage.setItem('tempType', JSON.stringify(tempType));
+  } else {
+    tempType = JSON.parse(localStorage.getItem('tempType'));
+    tempType[0] = 'Get Celsius';
+    fahCel.value = 'Get Fahrenheit';
+    fahCel2 = 'metric';
+    localStorage.setItem('tempType', JSON.stringify(tempType));
+  }
+};
+
+const changeBut = document.querySelector('.fahCel');
+changeBut.addEventListener('click', changeTemp);
+
+const temp = async (input) => {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${input}&appid=6828c8e901ce61eba78b5d02535eeed1&units=${fahCel2}`;
+
+  const grab = await fetch(url, {
```

Comment:
let us make request with error handling block, in this case you can use the `try and catch`. Errors in case of failure should be made visible to the user to make them aware of what is going on.